src/ufoLib2/objects/misc.py

```diff
@@ -70,19 +71,40 @@ def unionBounds(
 
 
 def _deepcopy_unlazify_attrs(self: Any, memo: Any) -> Any:
-    if getattr(self, "_lazy", True) and hasattr(self, "unlazify"):
+    if self._lazy:
         self.unlazify()
     return self.__class__(
         **{
             (a.name if a.name[0] != "_" else a.name[1:]): deepcopy(
                 getattr(self, a.name), memo
             )
             for a in attr.fields(self.__class__)
-            if a.init and a.metadata.get("copyable", True)
+            if a.init
         },
     )
 
 
+def _getstate_unlazify_attrs(self: Any) -> Tuple[Any, ...]:
+    if self._lazy:
+        self.unlazify()
+    return tuple(
```

Comment:
reminder to self if/when I pick this up again, upcoming attrs getstate now returns a dict, lo longer a tuple, see https://github.com/python-attrs/attrs/pull/1009

Reply:
done [5a8d9bc](https://github.com/fonttools/ufoLib2/pull/230/commits/5a8d9bc064b1b7e5c8166ab89234f90c42e327f4)